pages/k8s/release-notes.md

```diff
@@ -14,6 +14,432 @@ toc: False
 ---
 
 <!-- AUTOGENERATE RELEASE NOTES HERE -->
+# 1.29
+
+### February 12, 2024 - `charmed-kubernetes --channel 1.29/stable`
+
+The release bundle can also be [downloaded here](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.29/bundle.yaml).
+
+## What's new
+### aws-integrator
+Test storage and provider-id via integration testing
```

Comment:
Were you going to rationalise these comments or do you want me to do it?

Reply:
got 'em